OCR/frontend/e2e/ReviewTemplate.spec.ts

```diff
@@ -53,6 +53,6 @@ test.describe("ReviewTemplate Page", () => {
     const headers = page.locator("th");
     await expect(headers.nth(0)).toHaveText("Label"); // First header
     await expect(headers.nth(1)).toHaveText("Value"); // Second header
-    await expect(headers.nth(2)).toHaveText("Label Confidence"); // Third header
+    await expect(headers.nth(3)).toHaveText("Label Confidence"); // Third header
```

Comment:
is there a better way to select these headers, selecting by index is very brittle. maybe we should align on this practice as a team

Reply:
The only way I could come up with that would be slightly cleaner would be to have something like a loop: 

```
expectedHeaders = ['Label', 'Value', '', 'Label Confidence']
for (idx in expectedHeaders) {
  await expect(headers.nth(idx)).toHaveText(expectedHeaders[idx])
}
```
But I'm not sure that's less bad either

Reply:
yeah we should chat about that in the dev sync, we can add `data-testid` for better selectors, we can also select by the text, and make sure its a header.